packages/odyssey-react-mui/src/labs/NavRail/OktaAura.tsx

```diff
@@ -10,14 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import styled from "@emotion/styled";
 import { memo } from "react";
 
+const OktaAuraSvgComponent = styled("svg")(() => ({
```

Comment:
Copied over from the other PR
@bryancunningham-okta https://github.com/okta/odyssey/pull/2415#discussion_r1844484001
> @michaeltamaki-okta I think we can use the svg that currently lives in the `SideNav` folder. We can just wrap it in a styled component and apply these sizes to it. Rather than duplicate the SVG with a different size.

The SVG in the `SideNav` folder isn't being used (at least as far as I can tell), so I actually moved it to the `NavRail` folder here.